### DISCUSSION TITLE
申请加入文档维护组织

### DISCUSSION THREAD

**Discussion by wxsms** [vuejs/docs-next-zh-cn#779] [General] [answered]
既然已经成功加入，这里的个人信息就删除了。感谢各位的支持。

**Comment by Justineo** [comment]
支持，感谢你近期做了很多文档优化工作！

**Comment by Justineo** [comment]
@yyx990803 可否帮忙拉 @wxsms 进 [docs-cn team](https://github.com/orgs/vuejs/teams/docs-cn)?

**Comment by veaba** [comment]
@wxsms 最近一直做了很多文档优化的工作+及时同步文档内容。

**Comment by yyx990803** [comment, marked as answer]
已拉，感谢你的贡献！
